src/main/python/systemds/script_building/__init__.py

```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------------------
+#  Copyright 2020 Graz University of Technology
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ------------------------------------------------------------------------------
```

Comment:
Should the __init__ file be empty ?

Reply:
Yep, it can definitely be empty, and since we currently do not want anything public in this module it should be fine